pkcs11/src/new/types/session.rs

```diff
@@ -15,11 +15,15 @@ impl<'a> Session<'a> {
     pub(crate) fn handle(&self) -> CK_SESSION_HANDLE {
         self.handle
     }
+
+    pub(crate) fn client(&self) -> &Pkcs11 {
+        self.client
+    }
 }
 
 impl Drop for Session<'_> {
     fn drop(&mut self) {
-        if let Err(e) = self.client.close_session_private(self) {
+        if let Err(e) = self.close_private() {
```

Comment:
Where is this defined? `close_private`

Reply:
I renamed `close_session_private` to `close_private` since it's now implemented on a `Session`. I thought the `session` in the name was redundant 🤓

Reply:
Ah, yes, I forgot they were all moves to `Session`!